docs/4-functional-programming/fp-hometask/src/components/Sort/Sort.tsx

```diff
@@ -0,0 +1,52 @@
+import { FC } from 'react';
+import Radio from '@mui/material/Radio';
+import RadioGroup from '@mui/material/RadioGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import FormControl from '@mui/material/FormControl';
+import FormLabel from '@mui/material/FormLabel';
+
+import styles from './Sort.module.scss';
+import { SortValue, SortValueType } from 'src/types';
+
+interface SortProps {
+  setSelectedSort: (value: SortValue) => void;
+}
+
+// OR
+
+//interface SortProps {
+//  selected?: {};
+//  updateSelected?: (val) => void;
+//}
+
+// OR store can be global
+
+export const Sort: FC<SortProps> = ({ setSelectedSort }) => {
```

Comment:
Please, make this component controllable

Reply:
I don't quite understand what for ?))
Because now we have useEffect in App component and it controls everything
if we set value in Sort we should pass some callback for triggering changes of our dataForView, but how to control filtering in this case ?)

Reply:
From my experience, it is better to use controlled component here.

Reply:
done